src/main/java/com/summoner/lolhaeduo/domain/duo/service/DuoService.java

```diff
@@ -1,7 +1,35 @@
 package com.summoner.lolhaeduo.domain.duo.service;
 
+import com.summoner.lolhaeduo.common.dto.AuthMember;
+import com.summoner.lolhaeduo.domain.duo.entity.Duo;
+import com.summoner.lolhaeduo.domain.duo.repository.DuoRepository;
+import com.summoner.lolhaeduo.domain.member.enums.UserRole;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
 
 @Service
 public class DuoService {
+
+    private final DuoRepository duoRepository;
+
+    public DuoService(DuoRepository duoRepository) { // 생성자를 통해 주입
+        this.duoRepository = duoRepository;
+    }
+
+    public void deleteDuoById(Long duoId, AuthMember authMember) {
+        Duo existDuo = duoRepository.findById(duoId).orElseThrow(
+                () -> new IllegalArgumentException("듀오가 존재하지 않습니다.")
+        );
+
+        if (!authMember.getMemberId().equals(existDuo.getMemberId())) {
+            if (!authMember.getRole().equals(UserRole.ADMIN)) {
```

Comment:
- 해당 코드는 가독성이 떨어지는 것 같습니다ㅠㅠ

```suggestion
        if ( authMember.getRole().equals(UserRole.MEMBER) &&
            (authMember.getMemberId().equals(existDuo.getMemberId) )
```
-> 사용자이지만, 해당 듀오 신청자가 아니다

이렇게 변경해보는 건 어떨까요? 좀 더 직관적이지 않나요? :)